pkg/reconciler/taskrun/taskrun.go

```diff
@@ -155,6 +155,13 @@ func (c *Reconciler) ReconcileKind(ctx context.Context, tr *v1beta1.TaskRun) pkg
 		return c.finishReconcileUpdateEmitEvents(ctx, tr, before, err)
 	}
 
+	// Check for Pod Failures
+	failed, message := c.checkPodFailures(ctx, tr)
+	if failed {
+		err := c.failTaskRun(ctx, tr, v1beta1.TaskRunReasonImagePullFailed, message)
+		return c.finishReconcileUpdateEmitEvents(ctx, tr, before, err)
+	}
```

Comment:
nit - seems that `failed` and `message` are initialized to be used elsewhere so we can do this: 

```suggestion
	if failed, message := c.checkPodFailures(ctx, tr); failed {
		err := c.failTaskRun(ctx, tr, v1beta1.TaskRunReasonImagePullFailed, message)
		return c.finishReconcileUpdateEmitEvents(ctx, tr, before, err)
	}
```
https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow

Reply:
Thanks!!